framework/test/unit/modules/nft/method.spec.ts

```diff
@@ -248,24 +248,6 @@ describe('NFTMethod', () => {
 			);
 		});
 
-		it('should fail and emit Destroy event if NFT is escrowed', async () => {
```

Comment:
why was this test case removed ?

Reply:
Thanks for pointing it, reintroduced [here](https://github.com/LiskHQ/lisk-sdk/pull/8520/commits/dc86331f31ee014a3ddfc5338c5e4f2e7a2e9f23)